python/ray/tune/trainable.py

```diff
@@ -349,9 +359,13 @@ def stop(self):
         self._result_logger.close()
         self._stop()
 
-    def _train(self):
+    def _train(self, update=True):
```

Comment:
This should be evaluate=True right?

Reply:
OK, what about `validate=True`?

`evaluate` is not the right keyword... in most frameworks (keras, pytorch) `evaluate` is a separate concept than `train`, and `validate` is usually the (sub) concept tied to `train`